saleor/graphql/core/utils/__init__.py

```diff
@@ -36,6 +36,11 @@ def str_to_enum(name):
 
 def validate_image_file(file, field_name):
     """Validate if the file is an image."""
+    print(type(file))
```

Comment:
☠️

Reply:
Smells like a type hint would be welcome